modules/mono/mono_gd/gd_mono.cpp

```diff
@@ -260,32 +260,38 @@ using godot_plugins_initialize_fn = bool (*)(void *, GDMonoCache::ManagedCallbac
 #endif
 
 #ifdef TOOLS_ENABLED
-godot_plugins_initialize_fn initialize_hostfxr_and_godot_plugins(bool &r_runtime_initialized) {
-	godot_plugins_initialize_fn godot_plugins_initialize = nullptr;
-
-	HostFxrCharString godot_plugins_path = str_to_hostfxr(
-			GodotSharpDirs::get_api_assemblies_dir().path_join("GodotPlugins.dll"));
 
-	HostFxrCharString config_path = str_to_hostfxr(
-			GodotSharpDirs::get_api_assemblies_dir().path_join("GodotPlugins.runtimeconfig.json"));
+load_assembly_and_get_function_pointer_fn initialize_hostfxr(bool &r_runtime_initialized) {
+	String api_assemblies_config = GodotSharpDirs::get_api_assemblies_dir().path_join("GodotPlugins.runtimeconfig.json");
+	if (!FileAccess::exists(api_assemblies_config)) {
+		WARN_PRINT("Mono glue not found. Consider running the next commands from the Godot source root and recompiling using `mono_glue=yes`:\n\
+<godot_binary> --generate-mono-glue ./modules/mono/glue\n\
+./modules/mono/build_scripts/build_assemblies.py --godot-output-dir ./bin");
+		ERR_FAIL_V_MSG(nullptr, "Could not initialize hostfxr.");
+	}
 
-	load_assembly_and_get_function_pointer_fn load_assembly_and_get_function_pointer =
+	HostFxrCharString config_path = str_to_hostfxr(api_assemblies_config);
+	load_assembly_and_get_function_pointer_fn setup_assembly =
 			initialize_hostfxr_for_config(get_data(config_path));
-	ERR_FAIL_NULL_V(load_assembly_and_get_function_pointer, nullptr);
+	ERR_FAIL_NULL_V(setup_assembly, nullptr);
 
 	r_runtime_initialized = true;
-
 	print_verbose(".NET: hostfxr initialized");
+	return setup_assembly;
+}
 
-	int rc = load_assembly_and_get_function_pointer(get_data(godot_plugins_path),
+godot_plugins_initialize_fn initialize_godot_plugins(load_assembly_and_get_function_pointer_fn setup_assembly) {
+	ERR_FAIL_NULL_V_MSG(setup_assembly, nullptr, "Could not initialize Godot plugins due to hostfxr being uninitialized.");
+	godot_plugins_initialize_fn init_plugins = nullptr;
+	HostFxrCharString godot_plugins_path = str_to_hostfxr(GodotSharpDirs::get_api_assemblies_dir().path_join("GodotPlugins.dll"));
+	int rc = setup_assembly(get_data(godot_plugins_path),
 			HOSTFXR_STR("GodotPlugins.Main, GodotPlugins"),
 			HOSTFXR_STR("InitializeFromEngine"),
 			UNMANAGEDCALLERSONLY_METHOD,
 			nullptr,
-			(void **)&godot_plugins_initialize);
+			(void **)&init_plugins);
```

Comment:
I think the previous name was easier to understand, I feel like the new name is too vague.

Reply:
My reasoning for this variable name is the same one as the conversation above.